### DISCUSSION TITLE
SCRUM PREP MILESTONE 1-2

### DISCUSSION THREAD

**Discussion by Keiven-Soto** [uprm-inso4116-2024-2025-s1/semester-project-multiplayer-dominoes#87] [General]
This week i attended the weekly general meetings. Revised issues and communicated to the team about concerns of documentation tasks. Will be attending a weekly meeting with the professor and closed pull requests.

**Comment by PinkSylvie** [comment]
This week I was able to gather plenty of discussion with my questionaire for the documentation’s rough sketch. I also completed our team’s and webapp’s logo and icon.

**Comment by Adriana-Isabel** [comment]
This week, I worked on the Mock-up Webapp. The components of the web app are:

1. Header
> -	Home button
> - Instructions button
> - FAQs
> - Sing Up / Log In button

2. Logo

3. Leaderboard 

4. Game modes

5. Short Instructions
> - button for more precise instructions

6. Reviews / Feedback

7. Footer
> - Contact Us button
> - Social Media icons
> - FAQs button
> - Privacy Policy button
> - Logo

**Comment by AngelCIICMorales** [comment]
This week I organized a general team meeting and a team leader meeting. I assisted in the making of the Score metric system. In addition, I reviewed pull requests and oversaw the production of UI elements.

**Comment by Saul-Caban-UPRM** [comment]
Completed the assets for task #66 and organized a meeting with some of the team members from UI Team

**Comment by Negroni1** [comment]
Performed a proof of concept about WebSockets to have a better understanding of how to implement a multiplayer functionality for the game and to give team members consolidated information on how to implement them #61

**Comment by ebrain-mendoza121** [comment]
Made a base interface for creating and joining lobbies. Started working on documentation task on validation and verification

**Comment by fernandopizarro1** [comment]
Started and finished A basic AI opponent that can play against the player. It simply follows the game rules and plays any card that it can. Working on fixing a small game bug.

**Comment by sandy-marrero** [comment]
- Completed the Personas tasks and added the section to the documentation #48 
- Continued work on derived goals, consulted with the professor and I'm on the process of finalizing for review. #44 
- Created initial version of the Login/Signup form, waiting on UI/UX mockups so I can align my styling and send for review. #21 
- Working on a couple of documentation tasks that should be ready for review early next week.

**Comment by G4vp** [comment]
Successfully merged the main class for the table and domino logic, along with an experimental version of the game where players can place dominoes on the table and draw from the domino pool. This version demonstrates that the basic domino game rules are functional and provides other developers with a clear understanding of how to use the Table class.

**Comment by nsfrontera** [comment]
Started working on both my tasks, #67 and #69

**Comment by JadiellMV** [comment]
Attended weekly meetings, finished creating User Object (#84) and started working on implementing authentication endpoints (#17 ).

**Comment by FrancesC8** [comment]
Completed backend skeleton (#13). 
Waiting for others to design entity relationship diagrams to create the model classes in the backend (#28). 
Currently evaluating cloud hosting options for our apps and database (#94).

**Comment by Jvstinoo** [comment]
Continued work on #30 and started #73, will continue working on these during the weekend.

**Comment by frances-sepulveda** [comment]
This week I worked alongside @rukitoyordan to create the game board and four different avatars. Additionally, created power-ups that may be use if the player gets more than 3 doubles.

**Comment by ilikecheese117** [comment]
This week I made progress on the Save Game System by creating a class called "GameStatus" that saves the current table whenever the game is closed. Still want to work with it a bit more before creating a pull request for the branch.

**Comment by RicardoJRSs** [comment]
This week I made progress on the backend for the lobby system and started working on documentation for the project.

**Comment by AnthonyMed13** [comment]
Finished work for task Adding domain requirements #32 and task Adding needs to the documentation #19 still have not submitted it yet. Worked on task Implementing User Registration endpoints #18  and made some progress.

**Comment by DavidCuevas3** [comment]
Finished the wireframe for the Multiplayer Leaderboard

**Comment by print-Eruki** [comment]
This week I started the design for the ERD and the Architecture Diagram, when more in detail with other devs and teams to establish must-have columns and technologies being used. These two tasks will be finished next week.

**Comment by rukitoyordan** [comment]
This week I worked with @frances-sepulveda to draw in Dotpict four differing avatars: Blue, Red, Green and Yellow with somewhat gender-neutral appearance, as well as the domino board in cream with the Puerto Rican flag in the middle. Additionally worked on some documentation for M1.

**Comment by Joaquin939** [comment]
I talked to the AI team abou what rules they are going to use for the game so I could start working on my issue

**Comment by Gaby-Concepcion** [comment]
Milestone 1 (Week 2)

This week I worked on further developing and describing team dynamics and scenarios for the documentation. I also made progress on the tallying of points toward a player's score at the end of a game. Due to some exams I could not attend the weekly meeting and was delayed more than I'd hoped but for next week all that should be left is to push the implementations.

**Comment by ianramospr** [comment]
I'm working on the game overlays for when a player wins, loses, or it's their turn to play.
